static/app/views/eventsV2/table/columnEditCollection.tsx

```diff
@@ -34,6 +34,10 @@ type Props = {
   onChange: (columns: Column[]) => void;
   organization: Organization;
   className?: string;
+  noParameters?: boolean;
+  noHeaders?: boolean;
+  noTags?: boolean;
+  noEquation?: boolean;
```

Comment:
nit: can we name this to be more descriptive like `renderHeaders`, `renderTags` etc.? double negatives are kinda confusing. or instead of having 4 props here, should we just have one "source" prop saying it's for issue widget so don't render these? on one hand, it's 4 props but on the other i like having these generic props that can be passed down from any component

Reply:
Updated, went with the source prop here 👍